src/react-apps/applications/ux-editor/src/App.tsx

```diff
@@ -69,7 +69,7 @@ export class App extends React.Component<IAppComponentProps, IAppCompoentState>
 
       // Fetch form data
       formFillerActionDispatchers.fetchFormData(
-        `${altinnWindow.location.origin}/runtime/api/${reportee}/${servicePath}/Index/${instanceId}`);
+        `${altinnWindow.location.origin}/runtime/api/${servicePath}/${reportee}/${instanceId}`);
```

Comment:
@matsgm @lorang92 Is this in use now?i remember that it was mentioned that these url's with /runtime are not in use

Reply:
This part of the if/else is not in use. This is the old runtime "part" of Designer. Maybe it should be edited in another file?

Reply:
I'm pushing the refactoring of this file into this branch now.